src/logic/operate.js

```diff
@@ -0,0 +1,18 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let res;
+  if (operation === 'X') {
+    res = Big(numberOne) * Big(numberTwo);
+  } if (operation === '÷') {
+    res = Big(numberOne) / Big(numberTwo);
+  } if (operation === '+') {
+    res = Big(numberOne) + Big(numberTwo);
+  } if (operation === '-') {
+    res = Big(numberOne) - Big(numberTwo);
+  }
```

Comment:
- [ ] We still need to add another case for `%` operation (turns the number into percentage)

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png)